certs/localhost.local+1-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCpKDLAsIVmodqe
+WHeMynBMYek4bJJ/YQYhm4/r9Nks9jPApoBhaGHQks88Cmy+z4DPlNtPFBJOFQcb
+Dhjnf0rCrngMGgSBU6ROi5E2ZZ1VxrcrDD68+42DO4cSl5Q46wTp3dYa+K3lPyEN
+//UsVomvmPwyTCnGE6rnXl2gi+rGjxv8hsKkflFpCanA/Y6QN2j3O+3dhZrgJM3l
+oO+dLr1/5yoDlBgfcIyCzLHMUbMhnw0Ii5Vi/+daSZSMoOK2VdXDQ2HLE8Ih59Ps
+CprG08sTpodlFwx7CBukYPZ6UetYWoTTeRbQYhOiWxz/hq48Q721Q2ABqRnLniqT
+rNr8PwdnAgMBAAECggEAMxe8NK1crcJHeTp5aaIfx0JvoiZRHX5dzrrg/sEm88it
+mwvvjPxtRwWA6L6xV/TvEEQC9eC3IrSBP+/DH/E30JPIGMenArUT1NGdG9QXcLgj
+zTHWpVhJxxCs3LXCBGsgQc9Kgz0HsU2fG05rbGf7EHnh+wM8PuLj5RyApSR91lkX
+uwMfzVN4BlB3aj0zB6K2XoFP5yheVYdZ8k8zGWC4z9WvVuZcyTWPEkJ5spyZfpiZ
+ObHsD3IoN1ebZh4OgsUNppNLNEJgI5MteAe27AURQFJZyt77nEQHU2G+NyO4T1M4
+q67i5KL9ISbqGTSO56GXe5pAyFYZx7NezQHtvyiwGQKBgQDLxLY4bYdA8afRnuv+
+xIIe8/mMG2B8EoNZ6rsR4WCJC+Q848F5DgsuC5C9JGoKt0f1RTQeA8+khpkvijHe
+p+szhUT2XW6BQlibfqguRM8Idxr19/XIoHO00Jbbt5NTF3pkLtVJVok1aB99mf9/
+9I1z/0c4yfCmmaU8HDlMLqmvNQKBgQDUhEllpgbCiZxay2ZK12pbmSOLnXBbCBBl
+M9ncBOBzRbgpMEA95fN35L1zL9kezCkvPqoltf0x2loCKBZeMtB8MwppNa3Io9Hj
+Hxagcs9e//nREIXV2KT8is9mL1cNBVpduY66YXiGBo3HqsKum4I1ANrfa+EH5XYg
+CD9xWr/jqwKBgQClWEKweRfmN/EAJJ/ExDn92eUlrlVVeERDYzhW4vXNXuveCMXG
+cAHcZCAXioPNnDl0qaoHrPoBvAJzVATxcnpk/QAflWME2gxCaL0ZhlYkvVkQPBuh
+p6kcEKJJOaX/J671uhbyApkl0A61gY+UiaF6fauSRjLTTRuKMjDAQ6EG/QKBgBtd
+eXIMYfx06dxLKr3iZUCsRIZ63HfsXugZOCl7u0c8qA0tuGgAD0zV+Is3kxbDJV5/
+DDvU5mif/E0gyliQ5IXJsMKBdgJLDgDYBPreLdPG1G1WWlWCEySt3y0CVYxIry04
+cqePW4yulV0roOwI/EhE9uokIxL4OjC/vyOikukHAoGAIX9YBPyRRD5oWAPlpY3l
+JA44P5TAEqEYZkaPS0lrnE6hmEjzDDmXU9PI+yrGpMtbUt8oF31FoCqHlQX9K79z
+hto3x/sbF0lxWRQTLrx+Z5l6u/nwF6mMm0jsg29AolSHm7cD/lAVYOZVSpdNImer
+bvOgM4c33N7cxT1RnqNnUnc=
+-----END PRIVATE KEY-----
```

Comment:
Remove certificate binary and private key from repo

Reply:
It should be fixed now.